content/_index.md

```diff
@@ -29,7 +27,6 @@ To keep your site and services up and running we can also setup monitoring using
 popular tools such as Prometheus and Grafana.
 {{< /div >}}
 {{< div >}}
-
```

Comment:
nit: extraneous changes should generally be left out for diff readability (not that it matters, no need to change it now, just for future best practices this might be something we want to establish)

Reply:
Agreed, this was a bit clumsy, thanks for pointing it out.

Reply:
:+1: I end up doing this all the time too.